src/ai/backend/web/server.py

```diff
@@ -291,7 +291,12 @@ async def login_handler(request: web.Request) -> web.Response:
             'type': 'https://api.backend.ai/probs/generic-bad-request',
             'title': 'You have already logged in.',
         }), content_type='application/problem+json')
-    creds = await request.json()
+    secure_context = request.headers.get('X-BackendAI-Encoded', None)
```

Comment:
I check the value is string 'true'.
maybe it can be improved by a validator such as trafaret or something similar.

Reply:
Good point!